### ISSUE TITLE
Replies not getting sent

### ISSUE THREAD

**Issue by andrewlphilpott** [jariz/Noti#45]
I recently got a new computer and am having trouble getting my replies to SMS messages sent through Noti. I can click reply, enter my message, and send it, but it never actually gets sent. I thought maybe it was because I enabled encryption in both Noti and Pushbullet, but I just tried disabling that and I still can't reply to messages. Any idea what I could be doing wrong?

**Comment by andrewlphilpott** [comment]
Looks like other actions aren't working for me either. For example, deleting an email from a notification doesn't actually delete it.

**Comment by robertboullon** [comment]
same issue here, getting notifications but none of my responses send.

**Comment by quartz55** [comment]
Could this be related to encryption?

**Comment by robertboullon** [comment]
Yep weirdly enough I've found some success after turning on encryption and
then disabling it, now my notification responses work half the time if not
more.

**Comment by andrewlphilpott** [comment]
Interesting. I tried disabling encryption but I turned it back on after having my first reply fail.

**Comment by ABeltramo** [comment]
Have you a pro account? From what I see on [Pushbullet site](https://www.pushbullet.com/pro) only going pro will let you use mirror notification actions

**Comment by andrewlphilpott** [comment]
I did have a pro account when I was having this issue. I have since stopped using Pushbullet, so I’m not sure if this issue has been fixed or not.

**Comment by jariz** [comment]
reply != actions.
It's still part of the free plan (if you keep it below 100), so not really relevant afaik.

**Comment by jariz** [comment]
Can reproduce this, without pro & encryption enabled. (it does work with it disabled)
